WIP-CHANGELOG.md

```diff
@@ -13,3 +13,4 @@
 ### Changed
 
 - ClusterIssuers are used instead of Issuers
+- Updated velero chart to 2.15.0
```

Comment:
+ CRDs?

Reply:
You could also note which version of velero we have upgraded to

Reply:
Yea, I thought the CRDs as a part of the chart. But you are right, for us they are a separate thing.